R/developer.R

```diff
@@ -0,0 +1,162 @@
+#' @name developer_functions
+#'
+#' @title Developer functions for creating recipes steps
+#'
+#' @description
+#'
+#' This page provides a comprehensive list of the exported functions for
+#' creating recipes steps, and how to use them.
+#'
+#' @details
+#'
+#' # Creating steps
+#'
+#' [add_step()] and [add_check()] are required when creating a new step. It
+#' should be the other function call in your steps and should have the following
```

Comment:
"It should be the other function call" is ambiguous to me. Based on the shown code, maybe this could read "The output of `add_step()` should be the return value of all steps..."?

Reply:
fixed in [`fa88266` (#1163)](https://github.com/tidymodels/recipes/pull/1163/commits/fa8826634aed9ffb712174485ac8ee081ceb1f74)